app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -35,6 +40,22 @@ def host_products(host)
         host_subscription_facet(host)&.installed_products
       end
 
+      def host_products_names(host)
+        if products = host_products(host)
+          products.map(&:name).join(', ')
+        else
+          ''
+        end
+      end
+
+      def host_collections(host)
+        host.host_collections
+      end
```

Comment:
Is this really necessary?

Reply:
that's another disucssion, but given this file already has macros like this (`host_subscriptions` e.g.) I'd prefer consistency